client/a8c-for-agencies/components/items-dashboard/item-preview-pane/index.tsx

```diff
@@ -14,7 +14,7 @@ export const createFeaturePreview = (
 	id: string,
 	label: string | React.ReactNode,
 	enabled: boolean,
-	selectedFeatureId: string | undefined,
+	selectedFeatureId: string | undefined = '',
```

Comment:
I think we can revert this? We no longer call this function.

```suggestion
	selectedFeatureId: string | undefined,
```